ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/management-cvx.j2

```diff
@@ -0,0 +1,9 @@
+{# Management CVX #}
+{% if management_cvx is arista.avd.defined %}
+## Management CVX
+
+### Management CVX configuration
```

Comment:
Looks like the documentation is missing. I think we pushed wrong file. Please look into this.

Reply:
Need to generate artifacts after the template is updated.